### PR TITLE
dev-python/line_profiler: Version bump to 2.0

### PR DESCRIPTION
**Pull request by marbre**
@jlec Version bump to 2.0. I think we could than further drop 1.0-r1 with already included back ported patches now officially in 2.0 after the merge.

### CODE REVIEW

File: dev-python/line_profiler/line_profiler-2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="
+	dev-python/ipython[${PYTHON_USEDEP}]
```
Comment (SoapGentoo) [inline dev-python/line_profiler/line_profiler-2.0.ebuild]: so you only need ipython at build-time? not at runtime? you sure?

File: dev-python/line_profiler/line_profiler-2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	test? ( dev-python/pytest[${PYTHON_USEDEP}] )
+	"
+
+python_test() {
+	"${PYTHON}" -m unittest discover -v "${S}"/tests/ \
```
Comment (SoapGentoo) [inline dev-python/line_profiler/line_profiler-2.0.ebuild]: also prefer `EPYTHON` over `PYTHON`, `PATH`-dependent lookup is always better

File: dev-python/line_profiler/line_profiler-2.0.ebuild (hunk 1 above)
Comment (marbre) [inline dev-python/line_profiler/line_profiler-2.0.ebuild]: @SoapGentoo Thank for remark, you're absolutely right.

File: dev-python/line_profiler/line_profiler-2.0.ebuild (hunk 2 above)
Comment (marbre) [inline dev-python/line_profiler/line_profiler-2.0.ebuild]: Done.

**Pull request closed without merging**